vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/main/java/com/vaadin/flow/component/textfield/tests/PasswordFieldPage.java

```diff
@@ -115,12 +114,4 @@ private void addDisabledField() {
 
         add(passwordField, message);
     }
-
-    private void addInvalidCheck() {
```

Comment:
Can also remove the respective IT here I think: https://github.com/vaadin/flow-components/blob/cf78a2545f9bcbb96422896f29f1208b26239700/vaadin-text-field-flow-parent/vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/PasswordFieldPageIT.java#L142-L146

Reply:
Good catch, thanks! Done.